### PR TITLE
chore(license-book): update license book with herodevs

### PR DESCRIPTION
**Pull request by tasso94**
related to camunda/camunda-bpm-platform#4813

### CODE REVIEW

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 1]
```diff
@@ -1432,19 +1432,47 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 </pre></details></div>
-<div><details><summary><a href="https://github.com/xlts-dev/bower-angular">@xlts.dev/angular@1.9.3</a> (SEE LICENSE IN LICENSE.md)</summary><pre># The code in this repository is governed by two licenses
+<div><details><summary><a href="">@neverendingsupport/angularjs@1.9.7</a> (SEE LICENSE IN LICENSE.md)</summary><pre># This project contains IP governed by two licenses.
```
Comment (venetrius) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:1435]: 🙃 The link does not point to anywhere. 
The old link also does not point to a valid resource `https://github.com/xlts-dev/bower-angular` 

If it has any compliance or user value we could have a follow up task on this as well.

**Review by venetrius — APPROVED** [review]
👍

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md (hunk 1 above)
Comment (tasso94) [reply content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:1435]: Links are added automatically based on the respective `package.json` file. Better have no link than a broken link.

Comment (tasso94) [reply content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:1435]: I'm wondering if this makes our docs link checker fail 🤔

Comment (venetrius) [reply content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:1435]: We could nudge HeroDevs to provide an accessible license for their products and link it in their package.json.

**Pull request merged**